quicktests/overlaying/tests/basic/basic_area.js

```diff
@@ -12,10 +12,19 @@ function run(svg, data, Plottable) {
   // Will receive function arguments: (svg, data, Plottable)
   function getY(d) { return d.y; }
 
-  var dataseries = data[0].slice(0, 20);
-  var dataseries_top = data[1].slice(0, 20);
+  var deep_copy = function(from, to){
+    var deep_copy_xy = function(d){
+      to.push({'x': d.x, 'y': d.y});
+    };
+    from.forEach(deep_copy_xy);
+  };
 
-  for (var i = 0; i < 20; ++i) {
+  var dataseries = [];
+  deep_copy(data[0], dataseries);
+  var dataseries_top = [];
+  deep_copy(data[1], dataseries_top);
```

Comment:
Same as formatters.js, manipulations on the data require deep copies to be made instead of shallow copies, so that the two examples are identical.